numpy/distutils/exec_command.py

```diff
@@ -249,14 +249,25 @@ def _exec_command(command, use_shell=None, use_tee = None, **env):
         proc = subprocess.Popen(command, shell=use_shell, env=env,
                                 stdout=subprocess.PIPE,
                                 stderr=subprocess.STDOUT,
-                                universal_newlines=True)
+                                universal_newlines=False)
     except EnvironmentError:
         # Return 127, as os.spawn*() and /bin/sh do
         return 127, ''
+
     text, err = proc.communicate()
+    if sys.version_info >= (3, 0):
+        text = text.decode(sys.getdefaultencoding(), errors='replace')
```

Comment:
I think you can do this unconditionally, and then @pv's suggestion below will work (where the comment should actually be _"stdio uses bytes in python 2"_)